conf/application.conf

```diff
@@ -62,7 +62,7 @@ application.session.secure=false
 
 # The application languages
 # ~~~~~
-application.langs="en"
+play.i18n.langs = ["en"]
```

Comment:
In this file, can the following updates be done as this was updated?

- Can this play.filters.headers.contentSecurityPolicy on line 66 be updated to play.filters.csp.CSPFilter? See [here](https://github.com/playframework/playframework/blob/e5437eeceb8d75b70db87081bda3344274eb6123/web/play-filters-helpers/src/main/scala/play/filters/headers/SecurityHeadersFilter.scala) for more info. 

- Can the Secret key lines 52- 56 be removed since it is no longer needed? Thanks.

- Can play.http.filters can on line 43 also be removed please? (see [here](https://github.com/hmrc/bootstrap-play#configure-as-a-frontend-microservice) and [here](https://github.com/hmrc/bootstrap-play/blob/main/bootstrap-frontend-play-28/src/main/scala/uk/gov/hmrc/play/bootstrap/frontend/filters/FrontendFilters.scala) and also please update
uk.gov.hmrc.play.bootstrap.FrontendModule to uk.gov.hmrc.play.bootstrap.frontend.FrontendModule on line 42 as per [here](https://github.com/hmrc/bootstrap-play#configure-as-a-frontend-microservice) and [here](https://github.com/hmrc/bootstrap-play/blob/main/bootstrap-frontend-play-28/src/main/scala/uk/gov/hmrc/play/bootstrap/frontend/FrontendModule.scala)

- Can you please update uk.gov.hmrc.play.bootstrap.AuditModule to uk.gov.hmrc.play.audit.AuditModule since it has been migrated to play auditing [here](https://github.com/hmrc/play-auditing/blob/master/src-common/main/scala/uk/gov/hmrc/play/audit/AuditModule.scala). Also the comment on line 29 will need updating.
Thanks

Reply:
All good except for:
_Can this play.filters.headers.contentSecurityPolicy on line 66 be updated to play.filters.csp.CSPFilter? See [here](https://github.com/playframework/playframework/blob/e5437eeceb8d75b70db87081bda3344274eb6123/web/play-filters-helpers/src/main/scala/play/filters/headers/SecurityHeadersFilter.scala) for more info._

I tried but new CSP filters has completely different syntax and I am not 100% sure how to migrate these without breaking anything. I would be happy if we can create a jira to fix this as most of frontend repos has this in place.